pkg/operator/bootstrap.go

```diff
@@ -327,3 +370,28 @@ func getPlatformManifests(manifests []manifest, platformName string, lbType conf
 	}
 	return platformManifests
 }
+
+// Method to parse the contents of the `openshift-lb-config` configmap
+// passed in by the Installer.
+func readLBIPsFromConfigMap(fileData []byte) (string, string, error) {
```

Comment:
@sadasu Remember that these will be a comma separated list of strings.
```suggestion
func readLBIPsFromConfigMap(fileData []byte) ([]string, []string, error) {
```